src/App.tsx

```diff
@@ -67,6 +67,8 @@ function App() {
       <Routes>
         <Route path="/" element={<Navigate to={RoutePaths.vaults} replace={true} />} />
         <Route path={RoutePaths.vaults} element={<Honeypots />} />
+        <Route path={`${RoutePaths.vaults}/:pid`} element={<Honeypots />} />
+        <Route path={`${RoutePaths.vaults}/:pid/deposit`} element={<Honeypots showDeposit={true} />} />
```

Comment:
@leviyehonatan  Maybe it should be called `deposit-withdraw` since it leads to the deposit withdraw modal?
Or to have a path also to withdraw

Reply:
not now, this is enough